app/assets/javascripts/components/overview/available_actions.jsx

```diff
@@ -120,11 +120,11 @@ const AvailableActions = React.createClass({
       }
     // If user has no role or is logged out
     } else {
```

Comment:
This could change from an `else` to an `else if`, and then it won't add an extra level of nesting.